docs-client/src/containers/MethodPage/DebugInputs.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { ChangeEvent, Dispatch, useCallback, useReducer } from 'react';
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter';
+import json from 'react-syntax-highlighter/dist/esm/languages/hljs/json';
+import { Method, ServiceType } from '../../lib/specification';
+import { SelectOption } from '../../lib/types';
+import EndpointPath from './EndpointPath';
+import HttpHeaders from './HttpHeaders';
+import HttpQueryString from './HttpQueryString';
+import RequestBody from './RequestBody';
+import GraphqlRequestBody from './GraphqlRequestBody';
+
+SyntaxHighlighter.registerLanguage('json', json);
+
+interface OwnProps {
+  exactPathMapping: boolean;
+  exampleHeaders: SelectOption[];
+  supportedExamplePaths: SelectOption[];
+  exampleQueries: SelectOption[];
+  serviceType: ServiceType;
+  method: Method;
+  useRequestBody: boolean;
+  requestBody: string;
+  jsonSchemas: any[];
+  setRequestBody: Dispatch<React.SetStateAction<string>>;
+  additionalPath: string;
+  setAdditionalPath: Dispatch<React.SetStateAction<string>>;
+  additionalQueries: string;
+  setAdditionalQueries: Dispatch<React.SetStateAction<string>>;
+  additionalHeaders: string;
+  setAdditionalHeaders: Dispatch<React.SetStateAction<string>>;
+  stickyHeaders: boolean;
+  toggleStickyHeaders: Dispatch<React.SetStateAction<unknown>>;
+}
+
+const toggle = (prev: boolean, override: unknown) => {
+  if (typeof override === 'boolean') {
+    return override;
+  }
+  return !prev;
+};
+
+const extractUrlPath = (method: Method) => {
```

Comment:
Optional) I think this can be a utility method instead defined in a separate file for dedup